util/chplenv/chpl_libfabric.py

```diff
@@ -22,6 +22,8 @@ def get():
             error("CHPL_LIBFABRIC must not be 'none' when CHPL_COMM is ofi")
         elif chpl_platform.is_hpe_cray('target') and libfabric_val != 'system':
             warning('CHPL_LIBFABRIC!=system is discouraged on HPE Cray')
+
+        check_valid_var('CHPL_LIBFABRIC', libfabric_val, ['bundled', 'system'])
     else:
         libfabric_val = 'none'
```

Comment:
This case should print out an error if `overrides.get('CHPL_LIBFABRIC') != None`